### ISSUE TITLE
credit_card_validator has broken stripe-sdk

### ISSUE THREAD

**Issue by RaedJarrar** [ezet/stripe-sdk#111]
credit_card_validator has broken stripe-sdk. Please update the CC validator to the new syntax. Current work around used was to import: credit_card_validator: 1.1.0 in specific to make this work.

**Comment by RtypeStudios** [comment]
I'm seeing the same:

../../../Tools/flutter/.pub-cache/hosted/pub.dartlang.org/stripe_sdk-4.0.2/lib/src/models/card.dart:55:38: Error: Too few positional arguments: 2 required, 1 given.
      return _ccValidator.validateCVV(cvc, cardType: cardType).isValid;
                                     ^
../../../Tools/flutter/.pub-cache/hosted/pub.dartlang.org/stripe_sdk-4.0.2/lib/src/models/card.dart:57:38: Error: Too few positional arguments: 2 required, 1 given.

      return _ccValidator.validateCVV(cvc).isValid;
                                     ^

**Comment by RtypeStudios** [comment]
I think this could be easily fixed by limiting the upper version of credit_card_validator in the pubspec.yaml as a temporary solution.

Happy to create PR to do it?

**Comment by mariosemedo** [comment]
Workaround for now:

```
stripe_sdk:
    git:
      url: git://github.com/romme86/stripe-sdk.git
```

**Comment by ezet** [comment]
Fixed by #112

**Comment by JayPerfetto** [comment]
When will this be pushed to Pub.dev?
